tests/tests_unit/graph/test_loaders/test_dms_loader.py

```diff
@@ -70,7 +70,7 @@ def test_imf_attribute_nodes():
     knowledge_nodes = list(loader.load())
 
     assert len(knowledge_nodes) == 56
-    assert knowledge_nodes[0].sources[0].properties["predicate"].startswith("http")
+    assert knowledge_nodes[0].sources[0].properties["predicate"] == "CFIHOS-40000524"
```

Comment:
This is a result of the new change. All the IMF properties are AnyURI in the information model, which gets translated into text fields, and thus removes the namespace. 
![image](https://github.com/user-attachments/assets/214de5e0-4ad2-4582-9a1d-91b43c4f9004)

Open question: Should AnyURI instead translate into a connection with unknown source? In DMS, a direct relation that has not specified source.